spec/controllers/catalog_controller_spec.rb

```diff
@@ -99,6 +99,21 @@
       post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
       expect(response.body).to include(druid_no_pre)
     end
+    context 'can take two forms of druid input' do
```

Comment:
`can take druid with or without "druid:" prefix`  ?

As came up in devops today, think about reading this code without any context.  Like, me two months after I'm done with this project.  Or you, in a year.  Or ...